pkg/alertmanager/operator.go

```diff
@@ -146,12 +146,13 @@ func New(c prometheusoperator.Config, logger log.Logger) (*Operator, error) {
 
 func (c *Operator) RegisterMetrics(r prometheus.Registerer) {
 	c.statefulsetErrors = prometheus.NewCounter(prometheus.CounterOpts{
-		Name: "prometheus_operator_alertmanager_reconcile_errors_total",
+		Name: "prometheus_operator_reconcile_errors_total",
 		Help: "Number of errors that occurred while reconciling the alertmanager statefulset",
 	})
 
 	r.MustRegister(
 		c.statefulsetErrors,
+		c.triggerByCounterVec,
```

Comment:
nit: this should be in the next commit

Reply:
Are you talking about commit or PR? Because all commits will be squashed in the end. :blush:

Reply:
commit. cool as long as it's squashed then sounds good :)